### PR TITLE
feat(@wallet): speed up add account

### LINKED ISSUE #8571: On account creation only fetch current account data
When we add the account we reload the whole wallet, while we only need to fetch data for the newly created account

### PR DESCRIPTION
**Pull request by alaibe**
fixes #8571

status-go: https://github.com/status-im/status-go/pull/2997

### CODE REVIEW

File: src/app_service/service/wallet_account/service.nim [hunk 1]
```diff
@@ -180,7 +180,7 @@ QtObject:
         account.relatedAccounts = accounts.filter(x => not account.derivedFrom.isEmptyOrWhitespace and (cmpIgnoreCase(x.derivedFrom, account.derivedFrom) == 0))
         self.walletAccounts[account.address] = account
 
-      self.buildAllTokens()
+      self.buildAllTokens(toSeq(self.walletAccounts.keys()))
```
Comment (dlipicar) [inline src/app_service/service/wallet_account/service.nim:183]: Nit: `toSeq(self.walletAccounts.keys())` seems to be repeated in several places. Perhaps we could make it a function and give it a friendly name like `getWalletAddresses`

Comment (alaibe) [reply src/app_service/service/wallet_account/service.nim]: updated

**Comment by alaibe** [comment]
@dlipicar Have u updated the status-go repo with a make update when switching to that branch?
I can confirm it is working for me

**Comment by dlipicar** [comment]
> @dlipicar Have u updated the status-go repo with a make update when switching to that branch? I can confirm it is working for me

`make update` was not enough, but it started working after a `make clean`, my bad 🤦
It all looks good!

**Review by dlipicar — APPROVED** [review]
Everything looks good!

**Comment by anastasiyaig** [comment]
@alaibe i might found a regression there (wrong balance), mind checking? It works in master just fine.

Also ,please please resolve conflicts (sorry for testing delay)

Screenshot 2022-12-07 at 16 52 17

**Comment by alaibe** [comment]
image

Working after rebase

**Review by saledjenic — APPROVED** [review]

**Review by stefandunca — APPROVED** [review]

**Pull request merged**